src/components/Banner/index.js

```diff
@@ -1,27 +1,31 @@
 import AliceCarousel from "react-alice-carousel";
 import PropTypes from "prop-types";
-import { BannerContainer } from "./styled";
+import { BannerContainer, Img } from "./styled";
+import { StyledLink } from "../../styled";
 
-function Banner({ mainTitle, items }) {
+function Banner({ mainTitle, items, hasLink, href, searchParam }) {
   return (
     <BannerContainer>
       <h1>{mainTitle}</h1>
       <AliceCarousel disableButtonsControls>
-        {items.map(
-          (
-            {
-              id,
-              data: {
-                title,
-                name,
-                main_image: { url, alt },
-              },
-            },
-            index
-          ) => (
+        {items.map(({ id, slugs, data: { title, name, main_image } }, index) =>
```

Comment:
Remember to avoid deep destructuring in functions arguments. This would be better read as:

```javascript
{items.map(({ id, slugs, data }, index) => {
    const { title, name, main_image } = data;
    ...
```